Sources/Orbit/Support/Views/TileBorder.swift

```diff
@@ -1,5 +1,12 @@
 import SwiftUI
 
+public enum TileBorderStyle {
+    case none
+    case `default`
+    /// A card style that visually matches the iOS plain table section appearance.
```

Comment:
This mentions "card" - not sure if it makes sense without some context.

I guess it was there already...

Reply:
Fixed